app/models/maestrano/connector/rails/entity.rb

```diff
@@ -31,4 +37,20 @@ def self.creation_date_from_external_entity_hash(entity)
   def self.last_update_date_from_external_entity_hash(entity)
     entity['updated_at']&.to_time || Time.now
   end
+
+  private
+
+  def filter_connec_contacts(entities)
+    # Filter contacts to return a unique entry per email per shopify requirement
+    # Filter contacts to return people only
+    entities
```

Comment:
Considering we're using this almost as a 'service'  it could be extracted to the framework and made available for all connectors

Reply:
@MarcoBgn 
In Mailchimp it considers both people and organizations, whereas here it is only people. https://github.com/maestrano/connector-mailchimp/blob/develop/app/models/maestrano/connector/rails/entity.rb#L20

Would we use the one from mailchimp and in this case just call entities.delete_if {|e| e['resource_type'] == 'contacts' && !e['is_person'] } in get_connec_entities before calling the filter method?

Reply:
I believe the separation people/orgs was made cause at the time Contacts were not available. We could use a service object taking options, allowing for changes in behaviour depending on the connector that uses it. Something like:
`Maestrano::Connector::Rails::ContactsFilter.merge_by('email/address', opts)`
Definitely not a priority though

Reply:
Should I make a JIRA ticket for this?